npm/private/test/parse_pnpm_lock_tests.bzl

```diff
@@ -6,15 +6,15 @@ load("//npm/private:utils.bzl", "utils")
 def _parse_empty_lock_test_impl(ctx):
     env = unittest.begin(ctx)
 
-    parsed = utils.parse_pnpm_lock("")
+    parsed = utils.parse_pnpm_lock_yaml("")
```

Comment:
I think we need to test both `parse_pnpm_lock_*` properly and not pick&choose like this. Especially after https://github.com/aspect-build/rules_js/pull/1454

Reply:
Actually now (after scrolling more) I see they share the main implementation so most of the testing doesn't need to be done with both? Except parsing edge cases such as this one (empty files) and things like invalid yaml in https://github.com/aspect-build/rules_js/pull/1454